pkg/virtualKubelet/provider/provider.go

```diff
@@ -26,7 +26,7 @@ type LiqoProvider struct { // nolint:golint]
 
 	advClient     *crdClient.CRDClient
 	tunEndClient  *crdClient.CRDClient
-	homeClient    *crdClient.CRDClient
+	nntClient     *crdClient.CRDClient
```

Comment:
nntClient stands for?

Reply:
NamespaceNattingTableClient, too long :grinning:

Reply:
anyway, all the three local clients will be replaced by a single dynamic client ASAP